resources/languages-customization.yml

```diff
@@ -1,3 +1,10 @@
+C#:
+  configuration_files:
+    - ".csproj"
+    - "appsettings.json"
```

Comment:
This file could be named `appsettings.${SOMETHING}.json` as well.

Reply:
My understanding is that `appsettings*.json` files only contain application-specific runtime configuration. Some common examples are logging configuration and database connection strings (which provide a good hint to which database, if any, is being used). They aren't used at build time by the SDK.

Project dependencies are generally stored in the project file (`.csproj`, etc) with modern .NET (.NET Core, .NET >= 5). The build system reads it from there and uses nuget under the hood to download them.

AFAIK, `app.config` and `web.config` are older .NET files. They are used by classic ASP.NET which I am not familiar with at all. If alizer wants to support only modern .NET, it probably doesn't need to look at them.